crates/client/src/client/schema/chain.rs

```diff
@@ -249,16 +249,47 @@ include_from_impls_and_cynic! {
 
 #[derive(cynic::QueryFragment, Debug)]
 #[cynic(schema_path = "./assets/schema.sdl")]
-pub struct DependentCost {
+pub struct LightOperation {
     pub base: U64,
-    pub dep_per_unit: U64,
+    pub units_per_gas: U64,
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct HeavyOperation {
+    pub base: U64,
+    pub gas_per_unit: U64,
+}
+
+#[derive(cynic::InlineFragments, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub enum DependentCost {
+    LightOperation(LightOperation),
+    HeavyOperation(HeavyOperation),
+    #[cynic(fallback)]
+    Unknown,
```

Comment:
What's this used for?

Reply:
Each enum should have a fallback variant. It is a requirement from `cynic`